com.simutrans.Simutrans.metadata.xml

```diff
@@ -0,0 +1,41 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--Copyright 2024 The Simutrans Team-->
+<component type="desktop-application">
+  <id>com.simutrans.Simutrans</id>
+  <metadata_license>CC0-1.0</metadata_license>
+  <project_license>Artistic-1.0</project_license>
+  <name>Simutrans</name>
+  <summary>Open source transportation simulation game</summary>
+  <description>
+    <p>
+		Simutrans is a freeware and open-source transportation simulator. Your goal is to establish a successful transport company. Transport passengers, mail and goods by rail, road, ship, and even air. Interconnect districts, cities, public buildings, industries and tourist attractions.
+    </p>
+  </description>
+  <launchable type="desktop-id">simutrans.desktop</launchable>
```

Comment:
This should be com.simutrans.Simutrans.desktop if you use rename-desktop-file it will get updated automatically.